packages/shared/src/fallbackClerk.ts

```diff
@@ -0,0 +1,50 @@
+import type { ClerkStatus } from '@clerk/types';
+
+/**
+ * Creates a minimal fallback Clerk instance when the main clerk-js script fails to load.
+ * This fallback is designed to be as simple as possible - it only provides the essential
+ * properties needed to trigger error UI components and prevent crashes.
+ */
+export function createFallbackClerk() {
+  const fallbackClerk = {
+    status: 'error' as ClerkStatus,
+    loaded: false,
+  };
+
+  return new Proxy(fallbackClerk, {
+    get(target, prop) {
```

Comment:
should we add some prop to indicate this is the fallback client, or is `status === 'error' && loaded === false` sufficient?

Reply:
[cccbd66](https://github.com/clerk/javascript/pull/6261/commits/cccbd66f81395e0f6533f313390be06556955715)